crates/astria-sequencer/src/genesis.rs

```diff
@@ -13,6 +14,7 @@ pub(crate) struct GenesisState {
     #[serde(deserialize_with = "deserialize_address")]
     pub(crate) ibc_sudo_address: Address,
     pub(crate) native_asset_base_denomination: String,
+    pub(crate) ibc_params: IBCParameters,
```

Comment:
Please mention this in the PR message that the genesis file was changed.

Reply:
Already there?

Reply:
Yeah, there at the bottom in breaking changes. Didn't see 🙏